client/layout/index.jsx

```diff
@@ -78,6 +80,15 @@ class Layout extends Component {
 					.classList.add( `is-${ this.props.colorSchemePreference }` );
 			}
 		}
+		if ( config.isEnabled( 'nav-unification' ) ) {
+			window.addEventListener( 'scroll', scrollCallback );
```

Comment:
Could this be simplified to the following? 
```suggestion
			window.addEventListener( 'scroll', handleScroll );
```

Reply:
This was done so that we can effectively clear the event listener as discussed here https://github.com/Automattic/wp-calypso/pull/46689#issuecomment-718101323. I have already tried using the exported function `handleScroll` directly in vain. I may be doing something wrong though..